packages/marketplace/test/fixtures/index.ts

```diff
@@ -0,0 +1,95 @@
+import {ethers} from 'hardhat';
+import {ZeroAddress} from 'ethers';
+
+import {exchangeSetup} from './exchange';
+import {mockAssetsSetup} from './assets';
+
+async function deploy() {
+  const [deployer, admin, user, defaultFeeReceiver, user1, user2] =
+    await ethers.getSigners();
+
+  const {
+    RoyaltiesRegistryAsUser,
+    Royalties2981ImplMock,
+    RoyaltiesRegistryAsDeployer,
+    OrderValidatorAsAdmin,
+    OrderValidatorAsDeployer,
+    OrderValidatorAsUser,
+    OrderValidatorUpgradeMock,
+    ExchangeUpgradeMock,
+    ExchangeContractAsUser,
+    ExchangeContractAsAdmin,
+    protocolFeePrimary,
+    protocolFeeSecondary,
+    ExchangeContractAsDeployer,
+    AssetMatcherAsDeployer,
+    AssetMatcherAsUser,
+    TrustedForwarder,
+  } = await exchangeSetup();
```

Comment:
An alternative may be using just one variable to make it shorter:
``
const exchange = await exchangeSetip();
.......
return {...exchange, }
```